node-tests/blueprints/component-test-test.js

```diff
@@ -0,0 +1,235 @@
+'use strict';
+
+const fs = require('fs-extra');
+
+const blueprintHelpers = require('ember-cli-blueprint-test-helpers/helpers');
+const setupTestHooks = blueprintHelpers.setupTestHooks;
+const emberNew = blueprintHelpers.emberNew;
+const emberGenerateDestroy = blueprintHelpers.emberGenerateDestroy;
+const modifyPackages = blueprintHelpers.modifyPackages;
+
+const chai = require('ember-cli-blueprint-test-helpers/chai');
+const expect = chai.expect;
+
+const generateFakePackageManifest = require('../helpers/generate-fake-package-manifest');
+const fixture = require('../helpers/fixture');
+
+describe('Blueprint: component-test', function() {
+  setupTestHooks(this);
+
+  describe('in app', function() {
+    beforeEach(function() {
+      return emberNew();
+    });
+
+    describe('with ember-cli-qunit@4.1.0', function() {
```

Comment:
This is where you'll need to update to match what I did in #216, by:

- getting rid of `ember-cli-qunit` and `ember-cli-mocha` in favor of `ember-qunit@4.6.0` and `ember-mocha@0.16.0`
- dropping the *extra* checks for RFC 232 code by making the default fixtures *be* the RFC 232 code: we don't want to support the legacy test style (`moduleForComponent` and friends) *at all*